src/Command/TwoStepVerificationCommand.php

```diff
@@ -16,34 +16,23 @@
 use FOS\UserBundle\Model\UserManagerInterface;
 use Sonata\UserBundle\GoogleAuthenticator\Helper;
 use Sonata\UserBundle\Model\UserInterface;
-use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
-/**
- * NEXT_MAJOR: stop extending ContainerAwareCommand.
- */
-class TwoStepVerificationCommand extends ContainerAwareCommand
+class TwoStepVerificationCommand extends Command
 {
-    /**
-     * @var ?Helper
-     */
-    private $helper;
+    protected static $defaultName = 'sonata:user:two-step-verification';
 
-    /**
-     * @var ?UserManagerInterface
-     */
+    private $helper;
     private $userManager;
```

Comment:
Why did you remove the phpdoc ? We should keep it.

Reply:
Added again :) 

Do we need the docs to help with IDE?